Sources/StreamVideo/CallState.swift

```diff
@@ -235,8 +236,12 @@ public class CallState: ObservableObject {
             break
         case .typeCallRtmpBroadcastFailedEvent:
             break
-        case .typeCallSessionParticipantCountsUpdatedEvent:
-            break
+        case let .typeCallSessionParticipantCountsUpdatedEvent(event):
+            participantCount = event.participantsCountByRole
```

Comment:
can we check if we're not in the call here?

Reply:
I moved the logic to the CallController so we have access on the joined state.